RecoParticleFlow/PFProducer/plugins/kdtrees/KDTreeLinkerPSEcal.cc

```diff
@@ -233,18 +233,20 @@ void KDTreeLinkerPSEcal::searchLinks() {
 void KDTreeLinkerPSEcal::updatePFBlockEltWithLinks() {
   //TODO YG : Check if cluster positionREP() is valid ?
 
-  // Here we save in each track the list of phi/eta values of linked clusters.
+  // Here we save in each PS the list of phi/eta values of linked ECAL clusters.
   for (BlockElt2BlockEltMap::iterator it = target2ClusterLinks_.begin(); it != target2ClusterLinks_.end(); ++it) {
+    auto psElt = it->first;
+    auto ecalEltSet = it->second;
```

Comment:
`const auto&`, otherwise I think you might be making a copy of the `BlockEltSet`.

Reply:
Done.